internals/slo.py

```diff
@@ -97,3 +92,21 @@ def record_comment(
       return True
 
   return False
+
+
+def is_gate_overdue(gate, appr_fields, default_slo_limit) -> bool:
+  """Return True if a gate's review is overdue."""
+  if gate.requested_on is None or gate.responded_on is not None:
```

Comment:
```suggestion
  if gate.requested_on is None or gate.responded_on is None:
```
Checking None?

Reply:
Actually it means  `is not Nonne` that the gate has been responded?

Reply:
Yep, that's right.  If it is not none, then a reviewer already responded.